components/common-go/cgroups/cgroup.go

```diff
@@ -85,3 +87,41 @@ func ReadFlatKeyedFile(path string) (map[string]uint64, error) {
 
 	return kv, nil
 }
+
+func ReadPSIValue(path string) (PSI, error) {
+	file, err := os.Open(path)
+	if err != nil {
```

Comment:
If err doesn't mean that file doesn't exist, I recommend recording the error log.